src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2557,7 +2557,7 @@ internal void HideUnhideNamespace(bool hide, string library, string namespc)
             }
         }
 
-        private void AddZeroTouchNodesToSearch(IEnumerable<FunctionGroup> functionGroups)
+        internal void AddZeroTouchNodesToSearch(IEnumerable<FunctionGroup> functionGroups)
```

Comment:
made internal so it can be called from certain tests to fill the searchModel with ZT nodes.